public/js/controllers.js

```diff
@@ -51,7 +52,7 @@ angular.module("app.controllers", [ ])
                     var url = "/create_ins";
                     var parameter = JSON.stringify({
                         image_id: $scope.selectedImage.id,
-                        container_name: $scope.containerName,
+                        container_name: signInUsername + $scope.containerName,
```

Comment:
`signInUsername + ':' + $scope.containerName` is better for human read. Use `:` or other delimiter.